### PR TITLE
Fix/get note to sdk

### PR DESCRIPTION
**Pull request by matany90**

### CODE REVIEW

File: intezer_sdk/util.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    ignore_emojis = options.get('ignore_emojis', False)
+
+    note = (f'Intezer Analyze File Scan\n'
+            f'=========================\n\n')
+
+    if verdict == 'malicious':
+        emoji = get_emoji(verdict, ignore_emojis)
+        main_family, gene_count = get_analysis_family(analysis, ['malware', 'malicious_packer'])
+    elif verdict == 'trusted':
+        emoji = get_emoji(verdict, ignore_emojis)
```
Comment (davidt99) [inline intezer_sdk/util.py:24]: you can do it once after the if-else. You can also remove the flag from the function itself

File: intezer_sdk/util.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
+import collections
+import logging
+from typing import List
+from typing import Optional
+from typing import Tuple
+
+from intezer_sdk.analysis import Analysis
+
+logger = logging.getLogger(__name__)
```
Comment (davidt99) [inline intezer_sdk/util.py:9]: remove

Comment (matany90) [reply intezer_sdk/util.py]: Done

File: intezer_sdk/util.py (hunk 1 above)
Comment (matany90) [reply intezer_sdk/util.py]: done

File: intezer_sdk/util.py [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+        main_family, gene_count = get_analysis_family(analysis, ['application', 'library', 'interpreter', 'installer'])
+    elif verdict == 'suspicious':
+        main_family, gene_count = get_analysis_family(analysis, ['administration_tool', 'packer'])
+    else:
+        main_family = None
+        gene_count = None
+
+    note = f'{note}{emoji} {verdict.capitalize()}'
+
+    if verdict == 'suspicious' or verdict == 'unknown':
```
Comment (davidt99) [inline intezer_sdk/util.py:43]: `verdict in ('suspicious', 'unknown')

Comment (matany90) [reply intezer_sdk/util.py]: Done

File: examples/sentinel_one_integration.py [hunk 4]
```diff
@@ -297,6 +199,8 @@ def parse_argparse_args():
     parser.add_argument('-t', '--threat-id', help='S1 threat id', required=True)
     parser.add_argument('-sv', '--skip-ssl-verification', action='store_false',
                         help='Skipping SSL verification on S1 request')
+    parser.add_argument('-ne', '--no-emojis', action='store_true',
+                        help='Ignore emojis on notes')
```
Comment (davidt99) [inline examples/sentinel_one_integration.py:203]: Rename

**Pull request merged**